src/graphql/operations/leaderboards.ts

```diff
@@ -8,31 +9,35 @@ export default async function (parent, args) {
 
   checkLimits(args, 'leaderboards');
 
+  const ORDER_FIELDS = ['vote_count', 'proposal_count', 'user'];
+  const DEFAULT_ORDER_FIELD = 'vote_count';
+
   const fields = {
     user: 'EVMAddress',
     space: 'string',
     vote_count: 'number',
     proposal_count: 'number'
   };
   const whereQuery = buildWhereQuery(fields, 'l', where);
-  const defaultOrder = 'votesCount DESC, proposalsCount DESC';
 
-  const orderBy = Object.keys(fields).includes(args.orderBy)
+  const orderBy = ORDER_FIELDS.includes(args.orderBy)
     ? args.orderBy
-    : null;
+    : DEFAULT_ORDER_FIELD;
   let orderDirection = (args.orderDirection || 'desc').toUpperCase();
   if (!['ASC', 'DESC'].includes(orderDirection)) orderDirection = 'DESC';
 
+  const orderQuery = uniq([orderBy, ...ORDER_FIELDS])
```

Comment:
why to include other `ORDER_FIELDS` 🤔

Reply:
weird that it is taking more time without it :D

Reply:
Including all fields to avoid duplication, like when we have multiple voters with 1 vote and 0 proposal, we should still have a fallback field to sort with